Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM mambaorg/micromamba:1.5.6
+
+WORKDIR /model
+
+# Address warning about python debugger missing breakpoints
+# due to frozen modules being used
+ENV PYDEVD_DISABLE_FILE_VALIDATION=1
+
+COPY --chown=$MAMBA_USER:$MAMBA_USER . .
+
+RUN micromamba create -y -n claymodel --file conda-lock.yml && \
```

Comment:
Hmm, I'm getting this error when running `docker build . -t clay --platform linux/amd64` locally:

```
0.384 Transaction starting
79.51 critical libmamba Failed to create dir 'info'
79.51 error    libmamba Error opening for reading "/opt/conda/pkgs/cudnn-8.9.7.29-h092f7fd_3/info/index.json": No such file or directory
79.51 error    libmamba Error when extracting package: [json.exception.parse_error.101] parse error at line 1, column 1: syntax error while parsing value - unexpected end of input; expected '[', '{', or a literal
79.51 cudnn-8.9.7.29-h092f7fd_3.conda extraction failed
79.73 critical libmamba Found incorrect download: cudnn. Aborting
```

Not sure if it's because something changed when I re-locked the conda-lock.yml file manually when doing the merge from main at https://github.com/Clay-foundation/model/pull/166/commits/5be9fa2fdafdd2e54692687edd1c1b50db2c12fe#diff-63113c19c5d310b8e350b302f279e2297ba6faa9ac9c99c1c82e83e508447865

Reply:
Oh jeez - I'm getting a segmentation fault: 

```bash
2.612 Transaction starting
1441.7 qemu: uncaught target signal 11 (Segmentation fault) - core dumped
1441.7 bash: line 1:    10 Segmentation fault      micromamba create -y -n claymodel --file conda-lock.yml
------
Dockerfile:11
--------------------
  10 |
  11 | >>> RUN micromamba create -y -n claymodel --file conda-lock.yml && \
  12 | >>>     micromamba clean --all --yes
  13 |
--------------------
ERROR: failed to solve: process "/usr/local/bin/_dockerfile_shell.sh micromamba create -y -n claymodel --file conda-lock.yml &&     micromamba clean --all --yes" did not complete successfully: exit code: 139
```

Wondering if it's either related to a lack of available memory or installing CUDA in docker. 

I recall that when we installed the model libraries locally we had to install pytorch without CUDA - could the CUDA installation have made its way into the conda-lock file and causing issues?